commons/src/test/java/eu/itesla_project/commons/io/ComponentDefaultConfigTest.java

```diff
@@ -6,17 +6,17 @@
  */
 package eu.itesla_project.commons.io;
```

Comment:
Should be moved to config package to be consistent with ComponentDefaultConfig.java